services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java

```diff
@@ -62,6 +65,7 @@
 import org.exoplatform.services.log.Log;
 import org.exoplatform.services.rest.resource.ResourceContainer;
 
+// TODO: Auto-generated Javadoc
```

Comment:
Cleanup it

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/13/commits/e93ddc3e549cd35e3bb596f7c5c6aeb0fdc73878